src/eventHandlers/continuousIntegrationEnd/index.ts

```diff
@@ -92,10 +92,10 @@ export const continuousIntegrationEndHandle = async (
 
   const pullRequestsInformationPromises: Array<
     Promise<PullRequestInformation | undefined>
-  > = [];
-
-  for (const pullRequest of pullRequests) {
-    pullRequestsInformationPromises.push(
+  > = pullRequests.map(
```

Comment:
Looks like here there/s no need for pushing into the array in the loop, so I switched to map

Reply:
🤔 Does this even need to be done in a `const`? We could just map inside of `Promise.all()`, right?

Reply:
sure, updated 👍